code_samples/front/custom_query_type/config/packages/views.yaml

```diff
@@ -0,0 +1,22 @@
+ezdesign:
+    design_list:
+        my_design: [ my_theme ]
```

Comment:
is creating and assigning necessary? If not I think it just clouds this example.

Reply:
This part of the file will not be displayed in the doc, but it's included in the file so we can easily test the examples